### PR TITLE
Create own plants

### LINKED ISSUE #119: Create own plants
- Write manually or take a picture of seed bag
- Should be able to provide a sort for the plant

### PR DESCRIPTION
**Pull request by Peterkmoss**
Closes #119

### CODE REVIEW

File: app/src/main/java/dk/mifu/pmos/vegetablegardening/fragments/dialogs/CreatePlantDialogFragment.kt [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        setDatePickerListeners()
+        addOptionsToPlantSowing()
+        setSaveButtonListener()
+        setCancelButtonListener()
+
+        return binding.root
+    }
+
+    private fun setDataFromArgs() {
+        val argPlant = args.plant
```
Comment (miewetkefust) [inline app/src/main/java/dk/mifu/pmos/vegetablegardening/fragments/dialogs/CreatePlantDialogFragment.kt:44]: I don't understand why the fragment for creating new plants takes a plant as parameter?

File: app/src/main/res/layout/app_bar_main.xml [hunk 2]
```diff
@@ -9,15 +9,15 @@
     <com.google.android.material.appbar.AppBarLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:theme="@style/Theme.VegetableGardening.AppBarOverlay">
+        android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar">
```
Comment (miewetkefust) [inline app/src/main/res/layout/app_bar_main.xml:12]: What does this change in terms of layout/looks?

File: app/src/main/res/raw/plantdata.csv [hunk 3]
```diff
@@ -1,4 +1,4 @@
-;Kategori;Tidligste såning/plantning;Seneste såning/plantning;Plantes eller sås;Sædskifteinterval;Antal frø/planter pr. meter række;Sådybde;Afstand mellem rækkerne i cm;Gødningsniveau;Forventet høst per række
+;Kategori;Tidligste såning/plantning;Seneste såning/plantning;Plantes eller sås;Sædskifteinterval;Antal frø/planter pr. meter række;Sådybde i cm;Afstand mellem rækkerne i cm;Gødningsniveau;Forventet høst per række
```
Comment (miewetkefust) [inline app/src/main/res/raw/plantdata.csv:1]: Why did you add this but no remove "cm" from all the data? "Sådybde i cm: 4 cm" doesn't add any value

File: app/src/main/java/dk/mifu/pmos/vegetablegardening/fragments/dialogs/CreatePlantDialogFragment.kt (hunk 1 above)
Comment (Peterkmoss) [reply app/src/main/java/dk/mifu/pmos/vegetablegardening/fragments/dialogs/CreatePlantDialogFragment.kt:44]: Rename to CustomPlantDialogFragment

**Review by miewetkefust — APPROVED** [review]
